src/plugins/inspector/public/ui/inspector_panel.tsx

```diff
@@ -104,26 +122,8 @@ class InspectorPanel extends Component {
             </EuiFlexItem>
           </EuiFlexGroup>
         </EuiFlyoutHeader>
-        { this.renderSelectedPanel() }
+        <EuiFlyoutBody>{this.renderSelectedPanel()}</EuiFlyoutBody>
```

Comment:
Much better solution to use the EuiFlyoutBody 😉

Reply:
Agreed.